src/recursive_sorting/recursive_sorting.py

```diff
@@ -1,17 +1,66 @@
+l = [1, 5, 8, 4, 2, 9, 6, 0, 10, 3, 7]
+
+
 # TO-DO: complete the helpe function below to merge 2 sorted arrays
 def merge( arrA, arrB ):
     elements = len( arrA ) + len( arrB )
     merged_arr = [0] * elements
     # TO-DO
-    
+    print(f"elements: {arrA} and {arrB} - merged_arr {merged_arr}")
+    i = 0
+    y = 0
+    z = 0
+    while i < len(arrA) and y < len(arrB):
```

Comment:
This is a really optimal solution. It saves all the time of popping and appending to arrays, by replacing it with counters.